app/assets/templates/layout/navbar_right.html.slim

```diff
@@ -1,12 +1,12 @@
 ul.nav.navbar-nav.navbar-right
   li.my-account
-    a(ng-click="goToMyAccount()") My Account
+    a(ng-click="goToMyAccount()")= I18n.t 'sprangular.my_account'
```

Comment:
So you are correct,  right now this depends on a page reload switch locale's.   Since the Rails side has to be updated we are best to do a page reload.  And this way there is zero performance impact for those who only use 1 language.